src/ralph/assets/models/mixins.py

```diff
@@ -1,4 +1,6 @@
 # -*- coding: utf-8 -*-
+from datetime import datetime
```

Comment:
Maybe it's better to use:
https://docs.djangoproject.com/en/1.8/topics/i18n/timezones/#naive-and-aware-datetime-objects